cmd/utils/ont.go

```diff
@@ -305,7 +305,12 @@ func SignTransaction(signer *account.Account, tx *types.MutableTransaction) erro
 		tx.Payer = signer.Address
 	}
 	txHash := tx.Hash()
-	sigData, err := Sign(txHash.ToArray(), signer)
+	sink := common.NewZeroCopySink(nil)
+	sink.WriteBytes(txHash.ToArray())
```

Comment:
add function` Transaction.SigHashForChain(chainId)`  to avoid duplication.

Reply:
have done